packages/@jsii/dotnet-jsonmodel/.npmignore

```diff
@@ -1,7 +0,0 @@
-# by default, exclude everything
-*
-
-# bundle only...
-!*.nupkg
-!package.json
-!lib/index.js
```

Comment:
Does this need to be carried over?

Reply:
The other package already has that, since they were mostly copies of one-another from the boilerplate standpoint.